blockchain/projects.go

```diff
@@ -187,92 +157,104 @@ func (p *Projects) CreateInitialAccounts(projectID uuid.UUID) ([]*model.Account,
 		if err != nil {
 			return nil, err
 		}
-
-		account := model.AccountFromFlow(flowAccount, projectID)
-		account.Index = i
-		account.ID = uuid.New()
-		accounts[i] = account
 	}
 
-	return accounts, nil
+	return &p.accountsNumber, nil
 }
 
 // CreateAccount creates a new account and return the account model as well as record the execution.
-func (p *Projects) CreateAccount(projectID uuid.UUID) (*model.Account, error) {
```

Comment:
Probably need to change it back to account (won't keep commenting about the accounts)